### PR TITLE
Add flatland ros1

### PR DESCRIPTION
**Pull request by josephduchesne**
Please add the following dependency to the rosdep database.

## Package name:

flatland (metapackage), flatland_server, flatland_msgs, flatland_plugins, flatland_viz

## Package Upstream Source:

https://github.com/avidbots/flatland (? or n/a)

## Purpose of using this:

Flatland is lightweight 2d simulator (2.5d) built on Box2D and tightly integrated with ROS. It can serve as a gazebo alternative.

Distro packaging links:

## Links to Distribution Packages

n/a?

# Please Add This Package to be indexed in the rosdistro.

noetic
(a ros2 port is in mostly done works but one thing at a time).

# The source is here:

https://github.com/avidbots/flatland

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: noetic/distribution.yaml [hunk 1]
```diff
@@ -2662,6 +2662,23 @@ repositories:
...
+      packages:
+      - flatland
+      - flatland_msgs
+      - flatland_plugins
+      - flatland_server
+      - flatland_viz
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/avidbots/flatland-release.git
+      version: 1.3.1-2
```
Comment (nuclearsandwich) [inline noetic/distribution.yaml:2676]: Release stanzas in rosdistro are usually added by Bloom. Is there a reason that this was added manually?

**Comment by nuclearsandwich** [comment]
New package review:

- [x] At least one of the following must be present
  - [x] Top level license file: https://github.com/avidbots/flatland/blob/master/LICENSE
  - [ ] Per package license files
- [x] License is [OSI-approved](https://opensource.org/licenses): BSD-3-Clause
- [x] License correctly listed in package.xmls
  - [x] flatland: https://github.com/avidbots/flatland/blob/master/flatland/package.xml#L9
  - [x] flatland_msgs: https://github.com/avidbots/flatland/blob/master/flatland_msgs/package.xml#L6
  - [x] flatland_plugins: https://github.com/avidbots/flatland/blob/master/flatland_plugins/package.xml#L6
  - [x] flatland_server: https://github.com/avidbots/flatland/blob/master/flatland_server/package.xml#L6
- [x] Public source repo: https://github.com/avidbots/flatland/tree/master/
- [x] Source repository contains ROS packages and each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions
  - [x] flatland
  - [ ] flatland_msgs
  - [ ] flatland_plugins
  - [ ] flatland_server

**Review by nuclearsandwich — CHANGES_REQUESTED** [review]
I would prefer that the release stanza be added in a bloom PR rather than a manual PR. But without the release stanza this source entry looks good and passes new package review.

File: noetic/distribution.yaml (hunk 1 above)
Comment (josephduchesne) [reply noetic/distribution.yaml:2676]: My environment was poorly configured and was crashing when it went to push the diff after generating the release tags. I didn't want to generate a third release tag set so I just manually applied the diff. I'll have to figure out my environment one a dummy package sometime.

Comment (josephduchesne) [reply noetic/distribution.yaml:2676]: I'll fix the per package license files and try again tomorrow.

**Comment by josephduchesne** [comment]
Replaced with https://github.com/ros/rosdistro/pull/36035

**Pull request closed without merging**